modules/globebrowsing/rendering/chunkrenderer.cpp

```diff
@@ -112,6 +112,76 @@ ghoul::opengl::ProgramObject* ChunkRenderer::getActivatedProgramWithTileData(
     return programObject;
 }
 
+void ChunkRenderer::setCommonUniforms(ghoul::opengl::ProgramObject& programObject,
+                                      const Chunk& chunk, const RenderData& data)
+{
+    glm::dmat4 modelTransform = chunk.owner().modelTransform();
+    glm::dmat4 viewTransform = data.camera.combinedViewMatrix();
+    glm::dmat4 modelViewTransform = viewTransform * modelTransform;
+
+    if (_layerManager->layerGroup(
```

Comment:
If the short-circuit evaluation is not necessary, it might be simpler to define `const bool` before the if and make the if statement itself shorter and thus easier to understand.

Or:  Instead of `.size() > 0` use `!XX.empty()`